cmd/clusterawsadm/cmd/ami/list/list.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package copy
+package list
```

Comment:
This didn't have the correct package name so fixed it.

Reply:
This is strange, not sure how it was working like this.

Reply:
Ah this is the only file in the package.